Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -239,6 +244,22 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 	{
 		PrintMetricsTimer.SetTimer(10);
 	}
+	
+	if (StatStartFileTimer.HasTimerGoneOff())
+	{
+		USpatialNetDriver* SpatialDriver = Cast<USpatialNetDriver>(GetNetDriver());
+		if (ensure(SpatialDriver != nullptr))
+		{
+			FString FileName = FString::Format(TEXT("stat startfile {0}.ue4stats"), { SpatialDriver->Connection->GetWorkerId() });
```

Comment:
```suggestion
			FString Cmd = FString::Printf(TEXT("stat startfile %s.ue4stats"), *SpatialDriver->Connection->GetWorkerId());
```
- Cmd probably a more appropriate name 
- Printf does not require creating an argument array